src/test/mir-opt/simplify_try.try_identity.SimplifyLocals.after.mir

```diff
@@ -23,11 +24,14 @@ fn try_identity(_1: std::result::Result<u32, i32>) -> std::result::Result<u32, i
         }
     }
     scope 6 {
-        debug self => _0;                // in scope 6 at $SRC_DIR/core/src/result.rs:LL:COL
+        debug self => _2;                // in scope 6 at $SRC_DIR/core/src/result.rs:LL:COL
```

Comment:
This seems like a regression in debugging. Once that local is optimized out, there will be no more debug info for `self`.